### PR TITLE
Add code for a wizard step flow 1/c

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

This PR contains the work needed to stand up the wizard

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1970** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1970" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1970?utm_source=stack-comment).

File: src/components/Create/components/Wizard/Page.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
+import { ArrowLeftOutlined, ArrowRightOutlined } from '@ant-design/icons'
+import { Button, ButtonProps, Space } from 'antd'
+import { usePage } from './hooks'
+
+const BackButton = (props: ButtonProps) => {
+  return (
+    <Button size="large" {...props}>
```
Comment (tomquirk) [inline src/components/Create/components/Wizard/Page.tsx:7]: **suggestion**: perhaps we can use the `icon` prop for consistency.

File: src/components/Create/components/Wizard/Page.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      <ArrowLeftOutlined /> Back
```
Comment (tomquirk) [inline src/components/Create/components/Wizard/Page.tsx:8]: **suggestion**: trans <3

File: src/components/Create/components/Wizard/Page.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  return (
+    <Space direction="vertical" size="large" style={{ width: '100%' }}>
+      <div>
+        <h1>{title}</h1>
+        <p>{description}</p>
+      </div>
+      <div>{children}</div>
+      <div
+        style={{
+          marginTop: '16px',
```
Comment (tomquirk) [inline src/components/Create/components/Wizard/Page.tsx:73]: **suggestion**

might do the trick.

```suggestion
          marginTop: '1rem',
```

File: src/components/Create/components/Wizard/Wizard.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+const WizardContainer: React.FC = ({ children }) => {
+  return (
+    <div
+      style={{
+        margin: '0 auto',
+        width: '600px',
+        display: 'flex',
+        alignItems: 'center',
+        flexDirection: 'column',
+        padding: '32px 0',
```
Comment (tomquirk) [inline src/components/Create/components/Wizard/Wizard.tsx:17]:
```suggestion
        padding: '2rem 0',
```

File: src/components/Create/components/Wizard/Wizard.tsx [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    </div>
+  )
+}
+
+export const Wizard: React.FC<{ doneText?: string }> & {
+  Page: typeof Page
+} = props => {
+  const { currentPage, pages, goToPage } = useWizard({
+    children: React.Children.toArray(props.children),
+  })
```
Comment (tomquirk) [inline src/components/Create/components/Wizard/Wizard.tsx:30]: **praise** absolutely stellar job. I can see this working great. Nice one!!!

**Review by tomquirk — APPROVED** [review]
@wraeth-eth pre-approving!

File: src/components/Create/components/Wizard/Page.tsx (hunk 1 above)
Comment (wraeth-eth) [reply src/components/Create/components/Wizard/Page.tsx:8]: I think it is better to be consistent with our components here.

For the BackButton we want it showing on the left (which icon= in antd will give us) but NextButton, we want it on the right (antd will not give us this

**Pull request merged**